libs/@blockprotocol/type-system/rust/src/schema/data_type/closed.rs

```diff
@@ -6,7 +6,7 @@ use serde::{Deserialize, Serialize};
 use crate::{schema::DataType, url::VersionedUrl, Valid};
 
 #[derive(Debug, Clone, Serialize, Deserialize)]
-#[cfg_attr(target_arch = "wasm32", derive(tsify::Tsify))]
+// #[cfg_attr(target_arch = "wasm32", derive(tsify::Tsify))]
```

Comment:
I'm going to re-add this. It's disabled here because Tsify creates a `interface ClosedDataType extends DataType` with `DataType` being a `type`, not an interface, and extending a type is not allowed. We don't expose `ClosedDataType` from the graph currently anyway, so currently it's unused. I decided to deal with this later.